sdk/src/client/node_api/indexer/routes.rs

```diff
@@ -59,7 +59,7 @@ impl Client {
             .get_output_ids(&route, QueryParameters::empty(), true, false)
             .await?
             .first()
-            .ok_or_else(|| crate::client::Error::Node("no output id for alias".to_string()))?))
+            .ok_or_else(|| Error::NoOutput("alias"))?))
```

Comment:
I think it's fine to have multiple copies of the same error. We should be changing the error messages for wrapped errors to specify what component they are from anyway.

Reply:
I think this should rather be something like a `InvalidNodeResponse` error, because if the node responds with 200 this Vec should contain 1 output id, if not the node is doing something wrong

Reply:
Thats what i mean, 200 will be fine, and otherwise its 404, which will lead to NoOutput("alias") because it doesnt exist. We can easily just request a nonexistant one right?